.codeclimate.yml

```diff
@@ -20,6 +20,9 @@ plugins:
         enabled: false
       DeclarationOrder:
         enabled: false
+      NestingDepth:
+        enabled: false
```

Comment:
Now there is a `max_depth` of 3. I think there are reasons for allowing a bit deeper level of nesting. Cascade is part of CSS and there are occasions where we need to influence global styles. like for example this:

``` scss
.admin {
  &__section-header {
    .ofn-drop-down {
      &:hover,
      &.expanded {
         // I need to override this global styles
      }
  }
}
```

Another way of doing this is by using `!important`. But I guess nobody wants to go this way right? ;) Specificity is better than using `!important`

Reply:
I tried to setup a `max_depth` bigger than `3` (default). I tried `5` and `6`. It didn't work in this file `.codeclimate.yml` so I trieat `.scss-lint.yml` and I worked. But I realized this project is breaking this rule in a lot of places. Is just doesn't fail because some parts of the CSS is ignored by SCSS-Lint 😅 

![image](https://user-images.githubusercontent.com/49499/93664708-3a9f7a80-fa71-11ea-8f66-bb66de2182ba.png)

Reply:
LOL :rofl: 
I am convinced :-)
Is there a way to fix "some parts of the CSS is ignored by SCSS-Lint", that would be cool!
Thanks!!

Reply:
we might need to create an issue for this.